docs/how-to/jupyter-notebook.rst

```diff
@@ -83,6 +83,52 @@ Confirm the GPU is detected and usable by running:
 
    tf.config.list_physical_devices('GPU')
 
+Create an Intel GPU-enabled notebook
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create an Intel GPU-enabled Jupyter Notebook with IPEX or ITEX.
+
+.. note::
+
+   To launch an Intel GPU-enabled notebook, you must first :doc:`Enable Intel GPUs <../how-to/enable-gpus/enable-intel-gpu>`.
+
+To see the list of available Intel images, run:
+
+.. code-block:: bash
+
+   dss create --help | grep intel
+
+You should see an output similar to this:
+
+.. code-block:: bash
+
+        - intel-pytorch = intel/intel-extension-for-pytorch:2.1.20-xpu-idp-jupyter
+        - intel-tensorflow = intel/intel-extension-for-tensorflow:2.15.0-xpu-idp-jupyter
+
+Select one of them and create a notebook as follows:
+
+.. code-block:: bash
+
+   dss create my-itex-notebook --image=intel-tensorflow
+
+Confirm the devices are detected and usable by running:
```

Comment:
```suggestion
Confirm the GPU is detected and usable by running:
```
Let's focus on the GPU, not devices in general

Reply:
done